src/pages/interim/_love-trading.tsx

```diff
@@ -136,7 +138,7 @@ type RightCTASectionProps = LeftCTASectionProps & {
     button_props?: unknown
 }
 
-const RightCTASection = (params: RightCTASectionProps) => {
+const RightCTASection = <T extends object>(params: RightCTASectionProps) => {
```

Comment:
@niloo-fs  I didn't get this generic type here and then your type casting in line 154. would you please explain it?

Reply:
The type of button_props is **unknown** here and _**Spread types may only be created from object types.ts(2698)**_
for fixing that I used generic type!

Reply:
because I'm not sure about the things the may pass as a button_props here

Reply:
As far as I checked in the codebase, `button_props`  includes only one property which is `type` and it accepts string values.
these are the places that we are passing `buttom_props` to `LoveTradingComponent` : 
```
src/pages/interim/faq/_love-trading.tsx
src/pages/interim/dmt5/_love-trading.tsx
src/pages/interim/deriv/_love-trading.tsx
src/pages/interim/dbot/_love-trading.tsx
```

So rewriting `RightCTASectionProps` like this can fix the issue here: 

```
type RightCTASectionProps = LeftCTASectionProps & {
    button_props?: {
        type: string
    }
}
```

Can you check it please?

Reply:
Yes you are right @iman-fs I'll change this